src/components/ActionsPage.js

```diff
@@ -22,22 +22,22 @@ const Header = styled.div`
 `;
 
 export default class ActionPage extends Component {
-  
+
   share = () => {
     var url = "http://google.com";
     var text = "Replace this with your text";
-    window.open('http://twitter.com/share?url='+encodeURIComponent(url)+'&text='+encodeURIComponent(text), '', 'left=0,top=0,width=550,height=450,personalbar=0,toolbar=0,scrollbars=0,resizable=0');
+    window.open('http://twitter.com/share?url=' + encodeURIComponent(url) + '&text=' + encodeURIComponent(text), '', 'left=0,top=0,width=550,height=450,personalbar=0,toolbar=0,scrollbars=0,resizable=0');
   }
 
   render() {
     return (
       <ActionStyled>
         <Header>
-          <img src={'an image'} />
+          <img src={'an image'} alt={'some alt text'} />
```

Comment:
We need something better here, obviously.  To be addressed soon

Reply:
Yes, currently there are no images associated with Actions.